packages/core/src/actions/accounts/signMessage.ts

```diff
@@ -1,11 +1,11 @@
-import type { SignMessageReturnType } from 'viem'
+import type { SignMessageReturnType, Hex, ByteArray } from 'viem'
 
 import { ConnectorNotFoundError } from '../../errors'
 import { getWalletClient } from '../viem'
 
 export type SignMessageArgs = {
   /** Message to sign with wallet */
-  message: string
+  message: string | { raw: Hex | ByteArray }
```

Comment:
can we use `SignMessageParameters` from viem?

```suggestion
  message: SignMessageParameters['message']
```

Reply:
Should check that `SignMessageParameters` exists in this peer dep range `"viem": ">=0.3.35"`